addons/auth_totp/data/mail_templates.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="mail_template_totp_invite" model="mail.template">
```

Comment:
Could be great to set lang to ${object.lang} so that it is translated into recipient's language